src/core/background/scrobbler/listenbrainz-scrobbler.js

```diff
@@ -284,6 +284,14 @@ define((require) => {
 				trackMeta.additional_info.release_artist_name = song.getAlbumArtist();
 			}
 
+			if (song.getUniqueId() && song.metadata.label === 'Spotify') {
+				trackMeta.additional_info.spotify_id = song.getUniqueId();
```

Comment:
We have improved the metadata section of the listen and made them a bit more useful (see reference [here](https://listenbrainz.readthedocs.io/en/latest/users/json.html#brainzplayer-on-the-listenbrainz-website-playing-a-video-from-spotify))

The field `track_metadata.additional_info.origin_url` should also be added, in this case with the same value as the `spotify_id` field